### PR TITLE
Update event track projection handling 

### LINKED ISSUE #1898: Event tracks are not being built/destroyed properly on projection switch (UAT v3.0)
**Describe the bug**
This issue was observed after the fix to retain event tracks after projection switch in #1893. When an event track persists after switching projection, zooming in to that event track shows some unintended behavior with what track arrows/lines are being built and/or destroyed. 

**To Reproduce**
1. Go to `?v=-69.9102734375,-71.975234375,-47.8497265625,-57.244765625&t=2019-05-17-T00%3A00%3A00Z&e=EONET_2998,2019-05-17&l=MODIS_Terra_Sea_Ice(hidden),MODIS_Aqua_Sea_Ice(hidden),MODIS_Aqua_Brightness_Temp_Band31_Night(hidden),MODIS_Aqua_Brightness_Temp_Band31_Day(hidden),MODIS_Terra_Brightness_Temp_Band31_Night(hidden),MODIS_Terra_Brightness_Temp_Band31_Day(hidden),VIIRS_SNPP_DayNightBand_ENCC(hidden),Reference_Labels,Reference_Features,Coastlines(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor`
2. Switch to `antarctic` projection
3. Switch back to `geographic` projection
4. Zoom in to focused event and see extra arrows

**Screenshots**
For example, I clicked on an iceberg event in geographic, switched to antarctic, and back to geographic where I zoomed in some back and forth and moved the map slightly to get the following:
image

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by Benjaki2**
## Description
Updated the way event tracks handle proj changes. Now 3 instances are created at initialization and we toggle between on the 3 on proj change.

Fixes #1898
- [x] added deselection of point if not in projection as it works on production

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules (if applicable)

## Further comments
This one should be tested pretty thoroughly as it's a pretty significant change.

@nasa-gibs/worldview

### CODE REVIEW

**Comment by jasontk19** [comment]
Looks like tracks aren't loading when an event is indicated in a permalink.

Also, switching projections causes the track to disappear completely.

**Comment by Benjaki2** [comment]
Good finds @jasontk19 @minniewong. I think I got the fixes in. Please take another look when you all get a chance.

File: web/js/containers/sidebar/events.js [hunk 1]
```diff
@@ -153,7 +157,6 @@ const mapDispatchToProps = dispatch => ({
       dispatch(collapseSidebar());
     }
     if (dateStr) {
-      console.log(dateStr);
```
Comment (jasontk19) [inline web/js/containers/sidebar/events.js]: 👍

File: web/js/map/natural-events/track.js [hunk 2]
```diff
@@ -172,40 +151,46 @@ export default function naturalEventsTrack(ui, store, selectedMap) {
...
     }
+    self.active = true;
     self.trackDetails = newTrackDetails;
   };
 
-  var debounceTrackUpdate = lodashDebounce(
-    () => {
-      const selectedEvent = ui.naturalEvents.selected;
-      const map = ui.map.selected;
+  var debounceTrackUpdate = lodashDebounce(() => {
```
Comment (jasontk19) [inline web/js/map/natural-events/track.js]: 👍

File: web/js/modules/natural-events/selectors.js [hunk 3]
```diff
@@ -60,7 +60,6 @@ export function getEventsWithinExtent(
       isSelectedEvent &&
       olExtent.containsCoordinate(maxExtent, coordinates)
     ) {
-      console.log(maxExtent, coordinates);
```
Comment (jasontk19) [inline web/js/modules/natural-events/selectors.js]: 👍

**Comment by jasontk19** [comment]
Noting here that this also fixes #1905

**Pull request merged**